# 来源: https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-022-01509-7#:~:text=Genealogic%20studies%20and%20molecular%20analyses,years%20%5B2%2C%203%5D.

Skip to main content

Advertisement

Search

  * Explore journals 
  * Get published 
  * About BMC 
  * My account 



Search all BMC articles

Search

BMC Biology

  * Home
  * About
  * Articles
  * Submission Guidelines
  * Join the Editorial Board
  * Collections
  * Reviews
  * Submit manuscript




A nontuberculous mycobacterium could solve the mystery of the lady from the Franciscan church in Basel, Switzerland 

Download PDF

Download PDF

  * Research article
  * Open access
  * Published: 07 February 2023



# A nontuberculous mycobacterium could solve the mystery of the lady from the Franciscan church in Basel, Switzerland

  * Mohamed S. Sarhan ORCID: orcid.org/0000-0003-0904-976X1, 
  * Christina Wurst1, 
  * Alexandar Tzankov2, 
  * Andreas J. Bircher3,4, 
  * Holger Wittig5, 
  * Thomas Briellmann6, 
  * Marc Augsburger7, 
  * Gerhard Hotz8,9, 
  * Albert Zink1 &
  * …
  * Frank Maixner1

Show authors

_BMC Biology_ **volume 21**, Article number: 9 (2023) Cite this article

  * 4554 Accesses

  * 105 Altmetric

  * Metrics details




## Abstract

### Background

In 1975, the mummified body of a female has been found in the Franciscan church in Basel, Switzerland. Molecular and genealogic analyses unveiled her identity as Anna Catharina Bischoff (ACB), a member of the upper class of post-reformed Basel, who died at the age of 68 years, in 1787. The reason behind her death is still a mystery, especially that toxicological analyses revealed high levels of mercury, a common treatment against infections at that time, in different body organs. The computed tomography (CT) and histological analysis showed bone lesions in the femurs, the rib cage, and the skull, which refers to a potential syphilis case.

### Results

Although we could not detect any molecular signs of the syphilis-causing pathogen _Treponema pallidum_ subsp _. pallidum_ , we realized high prevalence of a nontuberculous mycobacterium (NTM) species in brain tissue sample. The genome analysis of this NTM displayed richness of virulence genes and toxins, and similarity to other infectious NTM, known to infect immunocompromised patients. In addition, it displayed potential resistance to mercury compounds, which might indicate a selective advantage against the applied treatment. This suggests that ACB might have suffered from an atypical mycobacteriosis during her life, which could explain the mummy’s bone lesion and high mercury concentrations.

### Conclusions

The study of this mummy exemplifies the importance of employing differential diagnostic approaches in paleopathological analysis, by combining classical anthropological, radiological, histological, and toxicological observations with molecular analysis. It represents a proof-of-concept for the discovery of not-yet-described ancient pathogens in well-preserved specimens, using de novo metagenomic assembly.

## Background

In 1975, a mummified corpse of a female individual was found in the Franciscan church (also known as the Barfüsser church) in Basel, Switzerland, during an excavation by the Archäologischen Bodenforschung Basel-Stadt in the church (Fig. 1A, B). The mummy’s coffin was encountered in a brick grave at a prominent position in the church (Fig. 1C, D), in front of the choir, along with another coffin that contained another human skeleton [1]. Genealogic studies and molecular analyses unveiled the mummy’s identity identifying her as Anna Catharina Bischoff (ACB, 29.03.1719-30.08.1787), a member of the upper class of post-reformed Basel, who died at the age of 68 years [2, 3]. By checking historical records of the church, it turned out that the mummy had been discovered earlier during the nineteenth century; then, due to ethical concerns, it has been reburied where it was again found in 1975 [2]. During this reburial, the coffin had been covered with soil (Fig. 1C).

**Fig. 1**

Description of the mummy’s finding site. **A** Map of Europe with zoom-in on Basel in Switzerland. **B** The Franciscan church during the renovation in 1975. **C** Photograph showing the first glance on the mummy tomb; recognizable are the overlapped hands. **D** The location of the burial chamber (indicated by red arrow) inside the Franciscan church (© Archäologische Bodenforschung, 1975/6 – Plan “Grabkammer und Grabschächte (G564)”, modified by H. Eichin 1981)

Full size image

The diseases and probable causes of death of ACB are still unknown, especially since the toxicological analyses of the mummy revealed high levels of mercury in different body organs [4], which is assumed to be the reason behind the mummification of the corpse (the antibacterial effect of mercury can slow down putrefaction), in addition to the dry conditions in the masonry grave shaft and the high salt concentration of the surrounding soils. Since mercury inhalation was a common treatment against infections, particularly syphilis [5], it was believed that she might have suffered from syphilis during her lifetime. This assumption was further supported by the computed tomography (CT) and histological results, which showed suggestive bone lesions in the skull, but degenerative lesions in femurs and rib cage. However, these lesions were histologically considered equivocal respecting late sequelae of syphilis infection, which led to additional differential diagnoses of a possible tuberculosis (TB) or Paget’s disease [6].

In this study, we aimed to investigate molecularly whether the mummy could have suffered from syphilis, by carrying out a comprehensive shotgun metagenomic analysis on different body organs, in order to detect possible DNA traces of the causing pathogen, _Treponema pallidum_ subsp. _pallidum_ , or other pathogens that might have led to the bone lesions or might be linked to the mercury treatment.

## Results

### Human mitochondrial DNA analysis of the samples confirms their origin

Initially, we analyzed the mitochondrial DNA of all tissues in order to confirm that they contain authentic DNA. All the samples taken from the mummy showed the same mitochondrial haplotype (i.e., U5a1+!16192) as previously reported [3], except for the skull bone sample, which showed additional background contamination with other human DNA (Additional file 1: Table S1) [3]. Interestingly, analysis of the human DNA of the maggots (Sample ID 3169), which were collected from underneath the mummy, revealed the same mitochondrial haplotype as of the mummy, indicating initial feasting on the decayed flesh of the mummy [7] or potential diffusion of mummy’s DNA into the surroundings [8]. The other control samples, coming from the skeletons of other individuals from the same tomb, showed two different mitochondrial haplotypes (Additional file 1: Table S1).

### Metagenomic analysis did not reveal any _Treponema_ genomes but unusual high prevalence of _Mycobacteriaceae_ in brain tissue

Driven by the radiological and histological observations and the toxicological analysis (Fig. 2A, B), we carried out a shotgun metagenomic screening of different samples from different body parts of the mummy, representing tissues where the infection can be expected to occur (please refer to Additional file 1: Table S1 for details). Although most of the tissues displayed overwhelming prevalence of postmortem microbial communities, e.g., Clostridia, we were still able to spot some tissue-specific taxa, particularly in the gut (_Turicibacter sanguinis_ and _Ruminococcus gnavus_) and the tooth (e.g., _Prevotella denticola_ and _Actinomyces dentalis_) samples (Additional file 1: Table S2). Additionally, we did not find any metagenomic reads assigned to the syphilis-causing pathogen _Treponema pallidum_ subsp _. pallidum_ (Additional file 2: Figure S1), nor even the containing-family _Spirochaetaceae_ , except for the tooth sample, which displayed presence of other _Treponema_ species, e.g., _T. socranskii_ , _T. denticola_ , and _T. maltophilum_ , which are all known to be linked to periodontitis and being members of the oral microflora [9, 10].

**Fig. 2**

Overview on the radiological, toxicological, and microbiological characteristics of the ACB mummy. **A** Computed tomography (CT)-based three-dimensional reconstruction of the skull. Notice the darker colors which represent lower x-ray densities than healthy bone. Copyright: Holger Wittig, Institute of Forensic Medicine, University of Basel. **B** Concentration of elemental mercury in different body samples, where the error bars refer to the standard errors. **C** Relative abundances of the top 12 microbial families on different body tissues as inferred by number of shotgun-metagenomic reads compared against the nr-database (please refer to the “Methods” section for details). Numbers in parentheses refer to sample IDs (please refer to Additional file 1: Table S1 for further details)

Full size image

Unexpectedly, the brain tissue displayed exceptional high abundance of the family _Mycobacteriaceae_ , representing more than 80% of the total microbial metagenomic reads (Fig. 2C, Additional file 2: Figure S2) [11,12,13]. In this respect, it is important to mention that the toxicological analysis displayed the highest mercury concentrations in the brain samples, i.e., up to 28 ng × mg−1 tissue material (Fig. 2B), since the brain is the target organ in the uptake or administration of elemental mercury. This opens a question on whether these two observations are correlated.

### De novo assembly revealed a nontuberculous mycobacterium (NTM) in the brain

Based on the high abundance of _Mycobacteriaceae_ in the brain, we performed a de novo metagenomic assembly on the brain metagenomic reads (for further information on assembly, please refer to the “Methods” section). We were able to resolve a high-quality metagenome-assembled genome (MAG, 99.5 % completeness and < 0.5 % contamination, as estimated by CheckM), belonging to a nontuberculous mycobacterium (NTM) species (Fig. 3A, B, Additional file 1: Table S3). Interestingly, more than 57% of the brain metagenomic reads were mapped against the assembled genome (Additional file 1: Table S4). The genome constituted of 66 contigs of total size of ~ 4.8 Mb and mean coverage of 185.9 ± 45.4 × (Additional file 1: Table S5).

**Fig. 3**

Genome-level taxonomic assignment of the brain bacterium. **A** Taxonomic assignment of the brain NTM contigs as assigned by searching against the NCBI-nt database, using BLASTn. The number of the assigned contigs is shown next the taxon names based on the lowest common ancestor (LCA) as determined by MEGAN (please refer to the “Methods” section for further details). **B** Unrooted phylogenetic tree of the family _Mycobacteriaceae_ , including a single representative genome of each species, based on PhyloPhlAn marker genes. The background colors of the clades refer to: red, _Mycobacterium_ spp.; yellow, _Mycolicibacter_ spp.; green, _Mycobacteroides_ spp.; and blue, _Mycolicibacterium_ spp. **C** Heatmap-based on MASH distances of all characterized species’ genomes within the genus _Mycolicibacterium_ including the genome of the brain bacterium (highlighted in bold red font). The heatmap annotations to the left of the heatmap refers to whether the microbe was isolated from a host or was reported as a human pathogen. For further details on the isolation sources, please refer to the Additional file 1: Table S6. **D** Damage plots of human DNA of different tissues as well as the brain NTM. The damage plots were generated considering the mapped reads of the indicated tissues different body tissues (i.e., tooth, intestinal tissues, skull, dura mater, and brain) against human genome (hg19) while the brain NTM was generated considering the brain metagenomic reads mapped against the brain NTM assembled genome. Ancient DNA damage represented by the terminal substitution of Cytosine to Thymine at the 5′ ends of the DNA fragments. The labels in parenthesis refer to sample ID and the mercury concentrations ± standard error. For further information on the read lengths distribution please refer to Additional file 2: Figure S3 [14, 15]. Note: The human DNA are showing variable levels of ancient DNA damages, despite of being of the same individual. The lowest levels of the human DNA damages are in the brain and dura mater samples, which goes with the abundance of the brain NTM and the mercury concentrations as well

Full size image

Considering that the genus _Mycobacterium_ has undergone major phylogenomic-based taxonomic reassignments and rearrangements [16, 17], we used species-representative genomes of the whole _Mycobacteriaceae_ family in order to gain an in-depth taxonomic characterization of our discovered genome within it, using PhyloPhlAn marker genes (please refer to the “Methods” section). The resulting phylogeny was highly congruent with the recently proposed taxonomy [16], having main clades representing the classical human pathogenic _Mycobacterium_ spp., _Mycobacteroides_ spp., _Mycolicibacter_ spp., and finally _Mycolicibacterium_ spp., where our genome falls within (Fig. 3B). Further, we compared our genome with all characterized species within the genus _Mycolicibacterium_ to find the closest relative within the genus. For this purpose, we carried out a pairwise genomic comparison between all species including ours, using the Mash distance tool [18]. The mash distances of the brain NTM genome is relatively distant from all the characterized species within the genus (minimum distance = 20.03%) and clusters close to _M. thermoresistibile_ , _M. hassiacum_ , _M. agri_ , and _M. moriokaense_ (Fig. 3C). Although the four species are known to be, like other members of the genus, environmental bacteria, they have previously been reported to cause infections in humans (Fig. 3C, and for detailed examples, please refer to Additional file 1: Table S6) [19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94]. Moreover, most of _Mycolicibacterium_ spp. have been previously isolated from hosts (Additional file 1: Table S6).

### Genome-wide analysis indicates potential virulence of the brain NTM

Before subjecting the brain genome to further functional analysis, we checked for the authenticity of this bacterium (henceforth referred to as “brain NTM”), i.e., being ancient on the one hand, and on the other being exclusively present in the brain, not being a contaminant from other tissues or even from the environment surrounding the burial site. Therefore, we tested the terminal deamination levels on the metagenomic brain reads mapped to the genome of the brain NTM (the “Methods” section). We noticed very low levels of terminal C-to-T and G-to-A substitutions, even less than the human DNA damage in this tissue (Fig. 3D, Additional file 2: Figure S3), although the fragment length distribution of the brain NTM was lower than of the human autosomal DNA (Fig. 3D, Additional file 2: Figure S3). When we further compared the human autosomal DNA from different organs, we realized variable levels of ancient DNA (aDNA) damages, correlating with the variable concentrations of mercury in different organs.

To further assess the possibility of the brain NTM being an external contamination, we investigated in addition to the mummy tissues more samples representing the following (Additional file 2: Figure S4) [2]: (i) bone and tooth samples from other skeletons found in the same burial site (upper coffin); (ii) textile sample and maggots that were found on the mummy; and finally, (iii) soil samples that were found covering the upper parts of the mummy. After mapping all metagenomic reads of each of the aforementioned samples against the brain NTM genome and considering threshold of minimum 3x coverage (please refer to the “Methods” section for details), we did not find any sufficient breadth (i.e., > 50% of the genome covered at least 3 times) for any of the mummy’s samples, except for the skull bone and dura mater samples, which appeared to carry the bacterium, having average breadth values of > 70% (Fig. 4A). Thereby, we excluded the possibility of the external contamination and continued with the functional analyses.

**Fig. 4**

Genetic map of the brain bacterium genome. **A** The heatmap shows distribution of the DNA of the brain NTM in the mummy’s samples as well as other control samples; The numbers to the left refer to the sample ID as referred to in Additional file 1: Table S1, while the letters refer to the samples group as follows: S, the mummy’s tissue samples; C1, control samples that were in contact with the mummy; C2, samples that were taken from other individuals from the same burial site; C3, soil sample that was collected from inside the coffin of the mummy (please refer to Additional file 1: Table S1 and Additional file 2: Figure S4 for further details). The presence is shown by breadth values with minimum coverage of 3x per site. The contigs are arranged in a descending order from left to right (for details on the contigs, please refer to Additional file 1: Table S5). The loci referred to in bold red fonts are gene clusters/operons and are further detailed in Additional file 2: Figure S5, while those in black bold are genes that are known to be involved in crossing blood brain barrier (BBB) and brain invasion. **B** The genetic map of the contig 38, which is assumed to be a phage genome, as inferred by the tool PHASTER (for further details on the phage annotation, please refer to Additional file 1: Table S7)

Full size image

We checked the overall virulence potential of the bacterium, by comparing all coding sequences (CDS) against the virulence factor database (VFDB) [95]. The genome contained three different clusters of type VII secretion systems (T7SSs), which are responsible for effector proteins in pathogenic and non-pathogenic mycobacteria, and help to survive in the host by evading the immune system [96]. Considering the genome of _M. tuberculosis_ as a reference, our genome contained the core genes of the ESX-1 system as well as the full genes of the ESX-3 and ESX-4, with the same exact synteny arrangement (Additional file 2: Figure S5).

Since the NTM genome was exclusively present in the brain, we checked for the ability of the bacterium to invade the brain and cross the blood brain barrier (BBB). Be and colleagues identified the genes Rv0311, Rv0931, Rv0986, and Rv0805 (CpdA) in _M. tuberculosis_ to be significantly involved in brain invasion and survival [97]. Indeed, we detected homologs of the four genes scattered throughout the genome (Fig. 4A). Moreover, it has been also previously reported that NTM can invade brain within circulating macrophages [98].

We additionally checked the brain NTM genome for the presence of toxin-antitoxin (TA) systems that are assumed to be contributing to the virulence of mycobacteria [99]. In comparison with other members of the genus _Mycolicibacterium_ , and representatives of other known pathogenic mycobacteria, we realized that the brain NTM genome together with _M. tuberculosis_ H37Rv were on the top in richness of TA systems (Additional file 2: Figure S6). Generally, TA systems are typically composed of a protein (toxin) and another antagonistic protein (antitoxin). Under stress conditions, the antitoxin, which is a labile molecule gets degraded, allowing the toxin molecule to halt essential cellular processes, leading to growth inhibition of bacteria [99]. Such process is reversible, which means that as soon as the conditions become favorable again, the antitoxin binds to the toxin and maintains essential cellular processes are restored. It is postulated that such TA systems play a great role in bacterial persistence under stress conditions, particularly in _M. tuberculosis_ infections, which sometimes mediates dormant states tolerant to antibiotic treatment and the host’s immune response, i.e., latent tuberculosis. Analysis of transcriptomic profiles of antibiotic-induced persistence in _M. tuberculosis_ revealed upregulation of 10 TA systems [100], which supports the assumptions of them being involved in pathogen persistence. In general, _M. tuberculosis_ and ACB’s brain NTM contain remarkably higher numbers of TA systems, compared with other mycobacteria, so we can deduce that it may have caused a persistent infection, by entertaining a privilege over other microbes in terms of survivability and competitiveness in extreme conditions.

Finally, and in relation to the high mercury concentrations in the brain, we checked for the mercury resistance genes in the genome of the brain NTM in comparison with the mercury resistance operon of the _M. marinum_ plasmid. We found in addition to the regulator protein gene and the core genes, i.e., mercuric reductase and alkylmercury lyase, other heavy metal resistance genes and antibiotic resistance genes (Additional file 2: Figure S5). The heavy metal resistance genes were mainly responsible for assimilation of copper, zinc, and arsenic, while the antibiotic resistance genes were directed against vancomycin, tetracycline, and beta-lactam antibiotics (Additional file 2: Figure S5).

### Evidence for a pre-infection transduction event that possibly increased the virulence

Although all mummy’s samples (except the brain, dura mater, and skull bones) seemed to be void of the brain NTM, they all displayed remarkable presence of one contig, i.e., contig 38, which indicates spread throughout the body, likely through the blood stream. To check the possibility that this contig might have been misplaced during the binning step, we went back to the metagenomic contigs and clustered them based on the coverage and the GC content, to whether this contig is an outlier to the rest of the genome. Considering both assembly methods, the contig was placed perfectly within the defined cluster of the brain NTM, showing similar coverage and GC content (Additional file 2: Figure S7). As further confirmation on the authenticity of the brain NTM, the contig 38 was completely absent from the samples of the other individuals and the soil sample. To explain the presence of this contig throughout the body, we hypothesized that it could be a circulating phage, since bacteriophages were shown to be present in the stool of patients with pulmonary diseases and to migrate through blood stream [101, 102]. Therefore, we screened for presence of viral sequences within our genome using the tool PHASTER (see the “Methods” section) and realized that the contig_38, which showed high prevalence in other mummy tissues, highly resembles phage genomes (Fig. 4B). Further, the CheckV tool displayed a high-quality viral genome with 91.35 completeness (Additional file 1: Table S8), having additionally other host/non-viral genes within the genome. By deeply annotating the contig, we found the basic structural proteins of phages, e.g., capsids, tail fibers, and portal proteins, in addition to other functional proteins, e.g., terminase, recombination proteins, and endolysin (Fig. 4B).

Interestingly, within this contig, there was the homologous gene of Rv0805 (CpdA), one of the genes that is involved in brain invasion and persistence. The presence of prophages has been recently reported in the genomes of clinical mycobacteria and assumed to aid in their virulence evolution. Moreover, when compared with environmental mycobacteriophages from PhagesDB, bacterial virulence genes were found to be enriched in the clinical mycobacterial prophages [103, 104]. This might indicate that the brain NTM has undergone a natural pre-infection transduction process that increased its virulence and enabled invasion and microbial survival in the brain.

## Discussion

The mummy of the Franciscan Church from Basel, Switzerland (ACB, 1719–1787) represents a unique example of multidisciplinary studied mummy [2]. Since the mummy is dated back to the eighteenth century, extensive historical, genealogical, and molecular investigations were necessary that resulted in reconstructing a family tree over 22 generations including nowadays living relatives [3]. The analyses showed that she belonged to a wealthy upper-class family, which might indicate that she might have had access to an advanced and special healthcare. The initial radiological and histological analyses of her mummified body indicated combination of different diseases, such as atherosclerosis and gallstones, and the toxicological analysis showed high concentrations of mercury in different body parts. Thus, studying the paleopathological status and the treatments she received provide insights into an important time in the European history, as she lived during the scientific revolution and before the onset of the industrial revolution.

One of the main intriguing findings was the high mercury concentration that has been revealed even at the mummy’s discovery time in 1975 [105] and was assumed to be one of the main reason for mummification. The presence of mercury also triggered some questions regarding her health status and in which context she had been exposed to these high concentrations of mercury. Historically, the beginning of mercury usage in the medical fields in Europe dates probably back to 1495, at the beginning of the first documented syphilis outbreak [106]. During that time, mercury had been already in use in the Arabic medicine to treat skin infections and leprosy [5, 107]. Therefore, it was adopted in Europe to treat syphilis and other infectious diseases by different methods of treatments, such as topical application, inhalation of mercury vapors, or even by ingestion of mercury salts [5, 108,109,110]. From the toxicological point of view, humans can get exposed to different forms of mercury, i.e., inorganic (e.g., mercury chloride), organic (e.g., methylmercury), or elemental (e.g., mercury vapors) [111]. The elemental form is absorbed by inhalation, passes from the lungs into the blood, crosses the blood-brain barrier and accumulates in the brain [112, 113]. While the inorganic forms, due to their lower liposolubility, were hypothesized to need to be accompanied with selenium (Se) that neutralizes the mercury’s toxicity to be able to access and cumulate in the brain [114]. On the other hand, mercury in its ionic forms is mainly accumulated in the kidney and liver. Considering that we found higher mercury concentrations in the brain than in the other body samples, we postulate that ACB was exposed to mercury vapors as a treatment for an extended time period. However, we cannot exclude that ACB was exposed to a treatment with an ointment containing colloidal elemental mercury or another inorganic mercury treatment, considering the correlation between the concentrations of the selenium and mercury, particularly in the brain of ACB (Additional file 2: Figure S8).

The classical explanation for the presence of mercury in the mummy would be the syphilis, which is mainly a sexually transmitted bacterial disease caused by _Treponema pallidum_ subsp. _pallidum_ [115]. Its symptoms are highly variable and can range from small chancres and nodular granulomatous skin lesion to late-onset prominent bone lesions [116]. Syphilis is also known as one of the “great imitators,” as it causes symptoms similar to other diseases making its diagnosis challenging [117]. For instance, in 1885, a bacillus bacterium was isolated from a syphilitic chancres [118, 119], which was assumed to be the causing pathogen, but later and by further research turned out to be the non-tuberculous _M. smegmatis_ that causes very similar ulcers to those of _T. pallidum_ subsp. _pallidum_ [77]. In our case, we could not find any metagenomic reads that can be assigned to _T. pallidum_ , and the only partially supporting clues would be the bone lesions unveiled by the radiological analysis with yet equivocal histopathology and the existence of mercury itself, where both are indirect evidence on syphilis. However, it might be useful in the future to consider employing a target enrichment capture approach particularly for those hardly to recover pathogens [120].

The other possible explanation for the mercury treatment can be deduced from the extraordinarily high abundance of _Mycobacteriaceae_ in the brain of ACB as well as scattered incidence in the dura mater and the skull bone samples. By employing de novo metagenomic assembly, we resolved a near-full NTM genome. In general, NTM are considered as environmental bacteria, which inhabit different niches like soils and water. Many of the NTM were reported to cause human infections but remained neglected for decades until their recent remarkable emergence in clinical infections [121, 122]. NTM affect primarily human lungs, causing pulmonary disease, but can also infect other body parts, particularly in immunocompromised patients [123]. In the ancient metagenomics and paleopathology fields, NTM are usually mentioned within environmental contexts or as contaminants, due to the difficulty to confidently assign them as true pathogens like _M. tuberculosis_ [124].

However, in this study, we opted to extensively analyze the genome of the brain NTM to explain their unusual existence and high abundance in the brain. The primary approach to check the DNA of a genome of whether it is ancient, is to analyze the ancient DNA damage pattern, i.e., terminal 5′C-to-T deamination. When we checked the damage pattern of the brain NTM, we found low damage levels (Fig. 3D). Interestingly, when we checked the damage of the human DNA of different organs of the mummy, we realized variability in the damage levels, negatively correlating with the mercury concentrations. For instance, the highest mercury concentrations were found in the brain samples, where we found the lowest human DNA damage (please refer to Fig. 3D for further examples). Despite of being a rare finding, this observation could explain the low damage levels on the brain NTM and highlights the inter-body variability of the DNA damages, which might open a discussion on using the ancient DNA damage as a primary tool for assessing modern contamination, particularly when dealing with unusually mummified materials.

In our case, the NTM contamination possibility can be toned down as soon as we consider the following: (i) the extraordinarily high abundance in the brain; (ii) the exclusive incidence in the brain and the close-by tissues and the absence from the other control samples as well as other organs of the mummy; (iii) the distribution of the putative phage within the mummy’s tissues and the absence from the control samples; (iv) the virulence potential of the genome particularly including genes that can enable crossing the BBB; (v) the richness of TA-systems that presumably enabled persistence under stress and unfavorable conditions; and, finally, (vi) the mercury-, heavy metal-, and antibiotic resistance genes of the bacterium. Therefore, we can gain confidence on the origin of the bacterium and its potential to be a brain pathogen that survived despite of the mercury treatment.

Overall, and based on the radiological, histological, toxicological, and molecular analyses of the ACB’s mummy, the current situation might suggest two different scenarios that explain her health status in relation to the high mercury concentrations. First, ACB suffered from syphilis and was exposed to a mercury treatment successfully, and _T. pallidum_ was completely eradicated, while the NTM brain infection occurred later. Second, she suffered from an NTM, and probably showed symptoms similar to syphilis, and was therefore subjected to mercury treatment, but the pathogen survived the mercury due to its content of mercury and heavy metal resistance genes. The first scenario can be supported by the radiological findings, i.e., the bone lesions of the skull and the ribs; however, it is not supported at the molecular level. While the second scenario is highly supported by the molecular analysis, and radiological signs could also support this hypothesis.

## Conclusions

In conclusion, this study spots the light on one NTM as one of the, not only nowadays, cause of neglected diseases and infections, which might have been misdiagnosed as syphilis or tuberculosis during the eighteenth century and may still be overlooked or misinterpreted nowadays in paleopathological studies due to the guided interest in more common and better-known diseases. The study of this mummy exemplifies the importance of employing differential diagnostic approaches in paleopathological analysis by combining classical anthropological and radiological observations with molecular investigations. It also demonstrates the value of employing comparative metagenomic approaches in analyzing multiple samples from one individual. Finally, the study shows the value of employing de novo metagenomic assembly in recovering extinct and not-yet described ancient genomes, in well-preserved specimens, and gaining better insights on their phylogenetic and functional characteristics that are often beyond the potential of other tools.

## Methods

### Mummy sampling

The analyzed tissues and bones have been sampled at the Natural History Museum of Basel (Additional file 1: Table S1). The sampled tissues were checked visually and histologically to confirm their tissue of origin. Additional control samples were taken from the soil surrounding the mummy and the textile found underneath the body. Moreover, bone and teeth samples were taken from the other skeletons found in another coffin in the same burial site (please refer to Additional file 2: Figure S4).

### DNA extraction, library preparation, and DNA sequencing

Amounts of 9–200 mg of the ancient materials or the control samples were used for DNA extraction, following the protocol described by Maixner and colleagues [125]. DNA extracts were quantified using QUANTUS (Promega, USA); then, 20 μl of each extract were converted into double-indexed DNA libraries following a special protocol for highly degraded ancient DNA [126]. All previous steps were carried out in the ancient DNA laboratory of the Eurac Research Institute for mummy studies in Bolzano, Italy. The double-indexed libraries were subjected to next generation DNA sequencing using HiSeqX (2 × 150 paired-end), resulting in sequencing depths of 9–51 × 106 paired-end reads per sample (Additional file 1: Table S1).

### Processing of ancient DNA sequences

We used the tool fastp [127] to trim the adapters and low-quality reads and to merge paired-end reads with at least 10 nucleotides overlap. Then, the quality filtered merged reads were de-duplicated and filtered for minimum sequence length of 25 nucleotides, using SeqKit [128].

### Human DNA analysis

To check the authenticity of the analyzed materials, we included, as a reference, in our analyses the shotgun metagenomic dataset generated from a tooth sample that was used to reveal the identity and the mitochondrial haplogroup of the mummy (https://www.ebi.ac.uk/ena/browser/view/PRJEB44723 ) [3]. For the analysis, we mapped the quality-filtered deduplicated merged reads against the human reference genome (build hg19) [129] and the human reference mtDNA genome (rCRS) [130] using Burrows-Wheeler Aligner (BWA) [131]. Then, we used SAMtools to filter for minimum mapping quality of 30, QualiMap to generate basic mapping statistics [132], and mapDamage 2.0 [14] to quantify the percentages of C-to-T and G-to-A substitution of the mapped ancient DNA reads. In the case of mitochondrial DNA, we additionally used option “--_rescale_ ” to rescale the quality scores of the damaged mis-incorporated sites and the tool Schmutzi to estimate the level of contamination based on deamination patterns [133]. For haplogroup assignment, we first converted the rescaled bam files into variant calling format (VCF) and then employed HaploGrep 2.0 [134].

Moreover, and to confirm the sex of the mummy samples, we used the mapped human DNA reads to compute the karyotype frequency of X and Y chromosomes, using a Maximum likelihood method [135].

### General microbial profiling

To have an overview on the microbial composition of the samples, we used the search tool BLASTX of DIAMOND v2.0.13 [136] to compare our metagenomic reads, with the default parameters, against the nr-protein database [137]. Then, we used MEGAN v6.21.16 [11] to process the DIAMOND outputs and to assign the metagenomic reads to their lowest common ancestor (LCA), with the parameters “ _\--minPercentIdentity 97_ ” and “ _\--minSupport 10_ ”. Since the BLASTX compares nucleotides against amino acid sequences, we confined our LCA filters to the family-level to minimize the false positives. The number and percentages of assigned reads are summarized in Additional file 1: Table S4.

### De novo assembly of metagenomic reads

We used the quality-filtered unmerged reads of the brain sample (Eurac ID 3045, 26411722 pair-end reads) to perform de novo sequence assembly, using the metagenomic assemblers MEGAHIT [138] and SPAdes with “--meta” option [139]. All contigs shorter than 1000 nt were excluded from the downstream analyses. The metagenomic binners MetaBAT2 [140], MaxBin2 [141], and CONCOCT [142] were used to resolve potential genomes, based on similarity of abundance and tetranucleotide frequency. Then, the DAS Tool was used to resolve consensus bins from each assembler, independently (Additional file 2: Figure S7).

Further, we checked the completeness and contamination of the resulting bins using CheckM [143] and kept only the high-quality genomes (completeness > 90% and contamination < 5%). CheckM checks for the presence of single-copy genes (SCG) specific for each lineage. If the program finds more than one copy of any of those SCG, it considers this as a potential contamination. To calculate the strain heterogeneity, the program calculates the relatedness between the multiple copies of those SCG, based on amino acid similarity. Then, the function “ _classify_wf_ ” of GTDB-Tk v1.5.0 (April 23, 2021) has been used, with default parameters, to classify the resulting bins taxonomically [144, 145].

As a result, two of the bins displayed identical taxonomic classification (g___Mycobacterium_ sp.) and MASH distance of < 0.01, each is resulting from different assembler, i.e., MEGAHIT and SPAdes. Therefore, we used SeqMan tool of DNASTAR [146] to reassemble the contigs of both bins, by aligning all contigs vs all contigs, in order to improve the assembly quality, e.g., N50 value and number of contigs. Finally, we ended up with near-full genome with 99.55% completeness and 0.45% contamination (Additional file 1: Table S3).

To check the taxonomy of the resulting contigs, we searched all the contigs against the NCBI-nt database [137]. Then, the module “ _blast2rma_ ” of MEGAN v6.21.16 [11] with the parameters “ _\--minPercentIdentity 80_ ” has been applied to assign the contigs to their lowest common ancestor (LCA).

### Phylogenetic analysis

To classify the resulting genome phylogenetically, we used the PhyloPhlAn 3.0 tool [147] to retrieve representative genomes of each species within the family Mycobacteriaceae. Then, we built the phylogeny based on the 400 universal PhyloPhlAn marker genes using the option “ _\--diversity low --accurate_ ”. The configuration file included the following tools with the default parameters set by PhyloPhlAn: DIAMOND v2.0.13 [136], MAFFT v7.427 [148], trimAl v1.4.1 [149], FastTree v2.1.11 [150], and RAxML v8.2.12 [151].

Based on the PhyloPhlAn phylogenetic assignment, we retrieved representative genomes of all characterized species belonging to the genus _Mycolicibacterium_ and added our brain bacterium genome, then calculated MASH distances all vs. all, using the “ _phylophlan_metagenomic_ ” module of PhyloPhlAn 3.0 [18].

### Checking the brain bacterium in the control samples

To check the presence of the brain bacterium in other tissues than the brain or in the control samples, we mapped the quality-filtered reads of each sample against the sequence of the brain bacterium, using Bowtie2 [152], with the option “ _\--end-to-end_.” Then, we sorted and indexed the resulted bam files using SAMtools [153], including a minimum mapping quality of 30. We used the script CMSeq (https://github.com/SegataLab/cmseq) to calculate the depth and breadth of the mapping. Positions were considered as true covered positions if they were covered at least 3 times. Afterwards, we calculated the breadth per contig and plotted the breadth as a heatmap (Fig. 4).

Finally, we used the tool DamageProfiler [15] to check for ancient DNA damage patterns, i.e., C-to-T and G-to-A substitutions, resulting from cytosine deamination.

### Bacterial genome annotation

In addition to the genomes of the _Mycolicibacterium_ spp., we included the genome of _Mycobacterium tuberculosis_ H37Rv from the Genbank databases (NCBI Reference Sequence: NC_018143.2) and _Mycobacteroides abscessus_ (NCBI Reference Sequence: NZ_CP034180.1), in the genome annotation analyses. We used the Prokka pipeline to annotate the genomes [154], implemented Prodigal for gene prediction [155], and RNAmmer to find ribosomal RNA genes [156].

To search for the virulence genes, we compared all coding sequences against the virulence factor database (VFDB) [95], using mmseqs2 with its default parameters. To annotate the type VII secretion system gene clusters, we manually extracted the regions from _M. tuberculosis_ H37Rv and our brain bacterium genome, based on Prokka annotation and compared them in a pairwise manner, using the BLASTp (all vs. all). Visualization of gene synteny was done using Clinker [157]. Following the same previous approach, we compared the mercury resistance operon in our brain bacterium genome to the well-characterized mercury resistance operon of _M. abscessus_.

To find the genes that are potentially involved in brain invasion [97], we manually retrieved the genes (_Rv0311_ ,_Rv0805_ , _Rv0931c_ , and _Rv0986_) from the Mycobrowser database (https://mycobrowser.epfl.ch). The tool OrthoFinder [158] was used to find the homologous sequences in our brain bacterium genome as well as the other _Mycolicibacterium_ spp. genomes, including the _M. tuberculosis_ and _M. abscessus_.

To check the presence of toxin/antitoxin (TA) systems in the analyzed genomes, we compared all coding sequences inferred by Prokka against the TA database (TADB 2.0), and confined the analysis to the experimentally validated type II TA loci [159]. We used mmseq2 for comparison using default parameters.

### Viral genome annotation

To search for viral sequences within the genome of the brain bacterium, we used the tool PHASTER (PHAge Search Tool Enhanced Release) [160], which predicts and annotates phage genes with comparison to curated phage and bacterial gene databases (https://phaster.ca). Additionally, we used the tool CheckV [161] to check the quality and completeness of the potential phage genomes. Then, we used different approaches to perform functional annotation of the contig which has been assigned as phages: (i) we used Prokka standard annotation as described previously; (ii) we used the PHASTER annotation tool; (iii) we used the tool MicrobeAnnoatator [162] implementing a DIAMOND search against different databases (i.e., KEGG Orthology, KO; Enzyme Commission, E.C.; Gene Ontology, GO; Pfam; and InterPro); and finally (iv) we used the MMseqs2 protein search tool [163] against the IMG/VR v3 database, which includes genomes of cultivated and uncultivated viruses [164].

### Mercury determination

The concentrations of mercury were measured in samples by inductively coupled plasma system coupled to mass spectrometry (ICP-MS; 7700 Series; Agilent, Palo Alto) at the University center of legal medicine (Geneva, Switzerland). Prior to analysis, samples were diluted with aqua regia to dissolve even poorly soluble mercury salts. The solution contained 10 ng/mL Rhodium (Rh) and 10 ng/mL Indium (In) as internal standards. In addition, each analytical batch of study samples was processed with laboratory controls, including method blanks and standard reference materials to continuously monitor method performance.

### Graphical plotting

The following R packages were used to plot the data: “ _ggplot2_ ,” “ _pheatmap_ ,” “ _webr_ ,” and “ _moonBook_ ” in R-studio (https://www.rstudio.com).

## Availability of data and materials

Raw sequencing data are publicly available on the European Nucleotide Archive (ENA, Project ID: PRJEB44723) [165].

## References

  1. Hotz G, Augsburger M, Briellmann T, Bircher A, Castella V, Fiechter R, et al. Der rätselhafte Mumienfund aus der Barfüsserkirche in Basel. Ein aussergewöhnliches Beispiel interdisziplinärer Familienforschung. Jahrbuch der Schweizerischen Gesellschaft für Familienforschung. 2018;2018:1–30.

Google Scholar

  2. Hotz G, Opitz-Belakhal C. Anna Catharina Bischoff. Die Mumie aus der Barfüsserkirche. Basel: Christoph Merian Verlag; 2021.

Google Scholar

  3. Wurst C, Maixner F, Castella V, Cipollini G, Hotz G, Zink A. The lady from Basel’s Barfusserkirche - molecular confirmation of the mummy’s identity through mitochondrial DNA of living relatives spanning 22 generations. Forensic Sci Int Genet. 2022;56:102604.

Article CAS Google Scholar

  4. Briellmann T, Hotz G, Augsburger M, Lenglet S. Toxikologische Untersuchungen. In: Hotz G, Opitz-Belakhal C, editors. Anna Catharina Bischoff die Mumie aus der Barfüsserkirche (Rekonstruktion einer Basler Frauenbiografie des 18 Jahrhunderts). Basel: Christoph Merian Verlag; 2021.

Google Scholar

  5. Tampa M, Sarbu I, Matei C, Benea V, Georgescu SR. Brief history of syphilis. J Med Life. 2014;7(1):4–10.

CAS Google Scholar

  6. Tzankov A, Bircher AJ. Mikroskopische Untersuchung. In: Hotz G, Opitz-Belakhal C, editors. Anna Catharina Bischoff die Mumie aus der Barfüsserkirche (Rekonstruktion einer Basler Frauenbiografie des 18 Jahrhunderts). Basel: Christoph Merian Verlag; 2021.

Google Scholar

  7. Njau DG, Muge E, Kinyanjui P, Omwandho C, Mukwana S. STR analysis of human DNA from maggots fed on decomposing bodies: assessment of the time period for successful analysis. Egypt J Forensic Sci. 2016;6(3):261–9.

Article Google Scholar

  8. Sarhan MS, Lehmkuhl A, Straub R, Tett A, Wieland G, Francken M, et al. Ancient DNA diffuses from human bones to cave stones. iScience. 2021;24(12):103397.

Article CAS Google Scholar

  9. You M, Mo S, Leung WK, Watt RM. Comparative analysis of oral treponemes associated with periodontal health and disease. BMC Infect Dis. 2013;13(1):1–13.

Article Google Scholar

  10. Verma D, Garg PK, Dubey AK. Insights into the human oral microbiome. Arch Microbiol. 2018;200(4):525–40.

Article CAS Google Scholar

  11. Huson DH, Beier S, Flade I, Gorska A, El-Hadidi M, Mitra S, et al. MEGAN Community edition - interactive exploration and analysis of large-scale microbiome sequencing data. PLoS Comput Biol. 2016;12(6):e1004957.

Article Google Scholar

  12. Lu J, Rincon N, Wood DE, Breitwieser FP, Pockrandt C, Langmead B, et al. Metagenome analysis using the kraken software suite. Nat Protoc. 2022:17:2815–39.

  13. Herbig A, Maixner F, Bos KI, Zink A, Krause J, Huson DH. MALT: fast alignment and analysis of metagenomic DNA sequence data applied to the Tyrolean iceman. bioRxiv. 2016:050559. https://www.biorxiv.org/content/10.1101/050559v1.

  14. Jónsson H, Ginolhac A, Schubert M, Johnson P, Orlando L. mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters. Bioinformatics. 2013;29(13):1682–4.

Article Google Scholar

  15. Neukamm J, Peltzer A, Nieselt K. DamageProfiler: fast damage pattern calculation for ancient DNA. Bioinformatics. 2021. https://academic.oup.com/bioinformatics/article/37/20/3652/6247758.

  16. Gupta RS, Lo B, Son J. Phylogenomics and comparative genomic studies robustly support division of the genus Mycobacterium into an emended genus Mycobacterium and four novel genera. Front Microbiol. 2018;9:67. https://doi.org/10.3389/fmicb.2018.00067.

Article Google Scholar

  17. Meehan CJ, Barco RA, Loh Y-HE, Cogneau S, Rigouts L. Reconstituting the genus Mycobacterium. Int J Syst Evol Microbiol. 2021;71(9):004922.

Article CAS Google Scholar

  18. Ondov BD, Treangen TJ, Melsted P, Mallonee AB, Bergman NH, Koren S, et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 2016;17(1):1–14.

Article Google Scholar

  19. Khan AA, Kim S-J, Paine DD, Cerniglia CE. Classification of a polycyclic aromatic hydrocarbon-metabolizing bacterium, Mycobacterium sp. strain PYR-1, as Mycobacterium vanbaalenii sp. nov. Int J Syst Evol Microbiol. 2002;52(6):1997–2002.

CAS Google Scholar

  20. Stanford J, Gunthorpe W. A study of some fast-growing scotochromogenic mycobacteria including species descriptions of Mycobacterium gilvum (new species) and Mycobacterium duvalii (new species). Br J Exp Pathol. 1971;52(6):627.

CAS Google Scholar

  21. Hennessee CT, Seo J-S, Alvarez AM, Li QX. Polycyclic aromatic hydrocarbon-degrading species isolated from Hawaiian soils: Mycobacterium crocinum sp. nov., Mycobacterium pallens sp. nov., Mycobacterium rutilum sp. nov., Mycobacterium rufum sp. nov. and Mycobacterium aromaticivorans sp. nov. Int J Syst Evol Microbiol. 2009;59(2):378–87.

Article CAS Google Scholar

  22. Cooksey RC, de Waard JH, Yakrus MA, Rivera I, Chopite M, Toney SR, et al. Mycobacterium cosmeticum sp. nov., a novel rapidly growing species isolated from a cosmetic infection and from a nail salon. Int J Syst Evol Microbiol. 2004;54(6):2385–91.

Article CAS Google Scholar

  23. Chamoiseau G. M. Farcinogenes agent causal du farcin du boeuf en Afrique. In: _Annales de Microbiologie_ ; 1973.

Google Scholar

  24. Toro A, Adekambi T, Cheynet F, Fournier P-E, Drancourt M. Mycobacterium setense infection in humans. Emerg Infect Dis. 2008;14(8):1330.

Article Google Scholar

  25. Gomila M, Ramirez A, Gasco J, Lalucat J. Mycobacterium llatzerense sp. nov., a facultatively autotrophic, hydrogen-oxidizing bacterium isolated from haemodialysis water. Int J Syst Evol Microbiol. 2008;58(12):2769–73.

Article CAS Google Scholar

  26. Greninger AL, Langelier C, Cunningham G, Keh C, Melgar M, Chiu CY, et al. Two rapidly growing mycobacterial species isolated from a brain abscess: first whole-genome sequences of Mycobacterium immunogenum and Mycobacterium llatzerense. J Clin Microbiol. 2015;53(7):2374–7.

Article CAS Google Scholar

  27. Gcebe N, Michel A. Gey van Pittius NC, Rutten V: comparative genomics and proteomic analysis of four non-tuberculous Mycobacterium species and Mycobacterium tuberculosis complex: occurrence of shared immunogenic proteins. Front Microbiol. 2016;7:795.

Article Google Scholar

  28. Schinsky MF, Morey RE, Steigerwalt AG, Douglas MP, Wilson RW, Floyd MM, et al. Taxonomic variation in the Mycobacterium fortuitum third biovariant complex: description of Mycobacterium boenickei sp. nov., Mycobacterium houstonense sp. nov., Mycobacterium neworleansense sp. nov. and Mycobacterium brisbanense sp. nov. and recognition of Mycobacterium porcinum from human clinical isolates. Int J Syst Evol Microbiol. 2004;54(5):1653–67.

Article CAS Google Scholar

  29. Okamori S, Asakura T, Nishimura T, Tamizu E, Ishii M, Yoshida M, et al. Natural history of Mycobacterium fortuitum pulmonary infection presenting with migratory infiltrates: a case report with microbiological analysis. BMC Infect Dis. 2018;18(1):1–6.

Article Google Scholar

  30. Shojaei H, Goodfellow M, Magee J, Freeman R, Gould F, Brignall C. Mycobacterium novocastrense sp. nov., a rapidly growing photochromogenic mycobacterium. Int J Syst Evol Microbiol. 1997;47(4):1205–7.

CAS Google Scholar

  31. Apajalahti JH, Kärpänoja P, Salkinoja-Salonen MS. Rhodococcus chlorophenolicus sp. nov., a chlorophenol-mineralizing actinomycete. Int J Syst Evol Microbiol. 1986;36(2):246–51.

CAS Google Scholar

  32. Poh M-E, Liam C-K, Ng K-P, Tan R. Mycobacterium brisbanense species nova isolated from a patient with chronic cavitary lung infection. Chest. 2014;145(4):858–60.

Article Google Scholar

  33. Franco MMJ, Paes AC, Ribeiro MG, de Figueiredo Pantoja JC, Santos ACB, Miyata M, et al. Occurrence of mycobacteria in bovine milk samples from both individual and collective bulk tanks at farms and informal markets in the southeast region of Sao Paulo, Brazil. BMC Vet Res. 2013;9(1):1–8.

Article Google Scholar

  34. Shahraki AH, Çavuşoğlu C, Borroni E, Heidarieh P, Koksalan OK, Cabibbe AM, et al. Mycobacterium celeriflavum sp. nov., a rapidly growing scotochromogenic bacterium isolated from clinical specimens. Int J Syst Evol Microbiol. 2015;65(Pt_2):510–5.

Article CAS Google Scholar

  35. Heidarieh P, Shojaei H, Hashemi A, Feizabadi MM, Daei-Naser A, Ataei B. First report of isolation of Mycobacterium elephantis from bronchial lavage of a patient in Asia. JRSM Short Rep. 2011;2(4):1–3.

Article Google Scholar

  36. Sethi S, Gupta V, Bhattacharyya S, Sharma M. Post-laparoscopic wound infection caused by scotochromogenic nontuberculous Mycobacterium. Jpn J Infect Dis. 2011;64(5):426–7.

Article CAS Google Scholar

  37. Allen DM, Chng HH. Disseminated Mycobacterium flavescens in a probable case of chronic granulomatous disease. J Inf Secur. 1993;26(1):83–6.

CAS Google Scholar

  38. Brown-Elliott BA, Wallace RJ Jr, Petti CA, Mann LB, McGlasson M, Chihara S, et al. Mycobacterium neoaurum and Mycobacterium bacteremicum sp. nov. as causes of mycobacteremia. J Clin Microbiol. 2010;48(12):4377–85.

Article Google Scholar

  39. Tsukamura M, Mizuno S, Gane N, Mills A, King L: Mycobacterium rhodesiae sp. nov. A new species of rapid-growing scotochromogenic mycobacteria. Japanese J Microbiol. 1971;15(5):407–16.

Article CAS Google Scholar

  40. Tortoli E, Kroppenstedt RM, Bartoloni A, Caroli G, Jan I, Pawlowski J, et al. Mycobacterium tusciae sp. nov. Int J Syst Evol Microbiol. 1999;49(4):1839–44.

Article CAS Google Scholar

  41. Jiménez MS, Campos-Herrero MI, García D, Luquin M, Herrera L, García MJ. Mycobacterium canariasense sp. nov. Int J Syst Evol Microbiol. 2004;54(5):1729–34.

Article Google Scholar

  42. Shojaei H, Daley C, Gitti Z, Hashemi A, Heidarieh P, Moore ER, et al. Mycobacterium iranicum sp. nov., a rapidly growing scotochromogenic species isolated from clinical specimens on three different continents. Int J Syst Evol Microbiol. 2013;63(Pt_4):1383–9.

Article CAS Google Scholar

  43. Chen Y-C, Jou R, Huang W-L, Huang S-T, Liu K-C, Lay C-J, et al. Bacteremia caused by Mycobacterium wolinskyi. Emerg Infect Dis. 2008;14(11):1818.

Article CAS Google Scholar

  44. Adékambi T, Stein A, Carvajal J, Raoult D, Drancourt M. Description of Mycobacterium conceptionense sp. nov., a Mycobacterium fortuitum group organism isolated from a posttraumatic osteitis inflammation. J Clin Microbiol. 2006;44(4):1268–73.

Article Google Scholar

  45. Kazda J. Mycobacterium sphagni sp. nov. Int J Syst Evol Microbiol. 1980;30(1):77–81.

CAS Google Scholar

  46. Lee SA, Raad II, Adachi JA, Han XY. Catheter-related bloodstream infection caused by Mycobacterium brumae. J Clin Microbiol. 2004;42(11):5429–31.

Article Google Scholar

  47. Dahl JL, Gatlin W III, Tran PM, Sheik CS. Mycolicibacterium nivoides sp. nov isolated from a peat bog. Int J Syst Evol Microbiol. 2021;71(3). https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.004438.

  48. Luis BAL, Díaz-Lomelí P, Gómez-Albarrán LP, Martínez-Gamboa A, Ponce-de-León A. Mycobacterium obuense bacteremia in a patient with pneumonia. Emerg Infect Dis. 2019;25(5):1015.

Article Google Scholar

  49. Marie I, Heliot P, Roussel F, Herve F, Muir J, Levesque H. Fatal Mycobacterium peregrinum pneumonia in refractory polymyositis treated with infliximab. Rheumatology. 2005;44(9):1201–2.

Article CAS Google Scholar

  50. Adékambi T, Foucault C, La Scola B, Drancourt M. Report of two fatal cases of Mycobacterium mucogenicum central nervous system infection in immunocompetent patients. J Clin Microbiol. 2006;44(3):837–40.

Article Google Scholar

  51. Xu C, Wu W, Pan H, Hui T, Wu Q, Zhou Z, et al. Mycobacterium Agri skin infection in a previously healthy patient: a case study. Infection Drug Resistance. 2021;14:2965.

Article Google Scholar

  52. Vuorio R, Andersson MA, Rainey FA, Kroppenstedt RM, Kämpfer P, Busse H-J, et al. A new rapidly growing mycobacterial species, Mycobacterium murale sp. nov., isolated from the indoor walls of a children’s day care Centre. Int J Syst Evol Microbiol. 1999;49(1):25–35.

Article CAS Google Scholar

  53. Balcázar JL, Planas M, Pintado J. Mycobacterium hippocampi sp. nov., a rapidly growing scotochromogenic species isolated from a seahorse with tail rot. Curr Microbiol. 2014;69(3):329–33.

Article Google Scholar

  54. Tsukamura M. Mycobacterium parafortuitum: a new species. Microbiology. 1966;42(1):7–12.

CAS Google Scholar

  55. Tsukamura M, Mizuno S, Toyama H. Mycobacterium pulveris sp. nov., a nonphotochromogenic mycobacterium with an intermediate growth rate. Int J Syst Evol Microbiol. 1983;33(4):811–5.

Google Scholar

  56. Kondo A, Mori K, Iwata J, Tamura M, Yamamoto T, Nakao Y, et al. Caseous necrotic granuloma in the pituitary stalk due to nontuberculous mycobacteria (Mycobacterium tokaiense) infection. Neurol Med Chir. 2006;46(2):80–3.

Article Google Scholar

  57. Singh J, Antony SJ. Prosthetic joint infection due to Mycobacterium moriokaense in an immunocompetent patient after a total knee replacement. In: _Baylor University medical center proceedings_ : Taylor & Francis; 2020. p. 97–9.

Google Scholar

  58. Tsukamura M, Mizuno S, Tsukamura S. Numerical analysis of rapidly growing, scotochromogenic mycobacteria, including Mycobacterium obuense sp. nov., nom. Rev., Mycobacterium rhodesiae sp. nov., nom. Rev., Mycobacterium aichiense sp. nov., nom. Rev., Mycobacterium chubuense sp. nov., nom. Rev., and Mycobacterium tokaiense sp. nov., nom. Rev. Int J Syst Evol Microbiol. 1981;31(3):263–75.

Google Scholar

  59. LéVy-FréBault V, Prome J-C, Grandry J, Boisvert H, David HL. Mycobacterium fallax sp. nov. Int J Syst Evol Microbiol. 1983;33(2):336–43.

Google Scholar

  60. Lee C-H, You H-L, Wang J-W, Tang Y-F, Liu J-W. Prosthetic joint infection caused by Mycobacterium alvei in an elderly patient. J Clin Microbiol. 2011;49(8):3096–8.

Article Google Scholar

  61. Ausina V, Luquin M, Barcelo MG, Lanéelle M, Lévy-Frébault V, Belda F, et al. Mycobacterium alvei sp. nov. Int J Syst Evol Microbiol. 1992;42(4):529–35.

CAS Google Scholar

  62. Huth RG, Brown-Elliott BA, Wallace RJ Jr. Mycobacterium mageritense pulmonary disease in patient with compromised immune system. Emerg Infect Dis. 2011;17(3):556.

Article Google Scholar

  63. Pettit AC, Jahangir AA, Wright PW. Mycobacterium doricum osteomyelitis and soft tissue infection. Emerg Infect Dis. 2011;17(11):2075.

Article Google Scholar

  64. Ballester F, Pujol I, Alcaide F, Pizarro I, Simó JM, Joven J, et al. First human isolate of mycobacterium poriferae in the sputum of a patient with chronic bronchitis. J Clin Microbiol. 2011;49(8):3107–8.

Article Google Scholar

  65. Casal M, Rey C. Mycobacterium gadium sp. nov. a new species of rapid-growing scotochromogenic mycobacteria. Tubercle. 1974;55(4):299–308.

Article CAS Google Scholar

  66. Trujillo ME, Velazquez E, Kroppenstedt RM, Schumann P, Rivas R, Mateos PF, et al. Mycobacterium psychrotolerans sp. nov., isolated from pond water near a uranium mine. Int J Syst Evol Microbiol. 2004;54(5):1459–63.

Article CAS Google Scholar

  67. Ballester F, Alcaide F, Pujol I, Hernández-Flix S, Simó JM, Joven J, et al. First human isolate of Mycobacterium madagascariense in the sputum of a patient with tracheobronchitis. Clin Chem Lab Med. 2013;51(2):e35–6.

Article CAS Google Scholar

  68. Kirschner P, Teske A, Schröder K-H, Kroppenstedt R, Wolters J, Böttger E. Mycobacterium confluentis sp. nov. Int J Syst Evol Microbiol. 1992;42(2):257–62.

CAS Google Scholar

  69. Adékambi T, Berger P, Raoult D, Drancourt M. rpoB gene sequence-based characterization of emerging non-tuberculous mycobacteria with descriptions of Mycobacterium bolletii sp. nov., Mycobacterium phocaicum sp. nov. and Mycobacterium aubagnense sp. nov. Int J Syst Evol Microbiol. 2006;56(1):133–43.

Article Google Scholar

  70. Adékambi T. Mycobacterium mucogenicum group infections: a review. Clin Microbiol Infect. 2009;15(10):911–8.

Article Google Scholar

  71. Shojaei H, Hashemi A, Heidarieh P, Hosseini N, Naser AD. Chronic pulmonary disease due to Mycobacterium monacense infection: the first case from Iran. Ann Lab Med. 2012;32(1):87–90.

Article CAS Google Scholar

  72. Tortoli E, Baruzzo S, Heijdra Y, Klenk H-P, Lauria S, Mariottini A, et al. Mycobacterium insubricum sp. nov. Int J Syst Evol Microbiol. 2009;59(6):1518–23.

Article CAS Google Scholar

  73. Zhang Y, Zhang J, Fang C, Pang H, Fan J. Mycobacterium litorale sp. nov., a rapidly growing mycobacterium from soil. Int J Syst Evol Microbiol. 2012;62(Pt_5):1204–7.

Article CAS Google Scholar

  74. Zhang D-F, Chen X, Zhang X-M, Zhi X-Y, Yao J-C, Jiang Y, et al. Mycobacteriumsediminis sp. nov. and Mycobacteriumarabiense sp. nov., two rapidly growing members of the genus Mycobacterium. Int J Syst Evol Microbiol. 2013;63(Pt_11):4081–6.

Article CAS Google Scholar

  75. Kim B-J, Kim J-M, Kim B-R, Lee S-Y, Kim G, Jang Y-H, et al. Mycobacterium anyangense sp. nov., a rapidly growing species isolated from blood of Korean native cattle, Hanwoo (Bos taurus coreanae). Int J Syst Evol Microbiol. 2015;65(Pt_7):2277–85.

Article CAS Google Scholar

  76. Tran PM, Dahl JL. Mycobacterium sarraceniae sp. nov. and Mycobacterium helvum sp. nov., isolated from the pitcher plant Sarracenia purpurea. Int J Syst Evol Microbiol. 2016;66(11):4480–5.

Article CAS Google Scholar

  77. Gordon RE, Smith MM. Rapidly growing, acid fast bacteria I: species’ descriptions of Mycobacterium phlei Lehmann and Neumann and Mycobacterium smegmatis (Trevisan) Lehmann and Neumann. J Bacteriol. 1953;66(1):41–8.

Article CAS Google Scholar

  78. Hormisch D, Brost I, Kohring G-W, Giffhorn F, Kroppenstedt R, Stackebradt E, et al. Mycobacterium fluoranthenivorans sp. nov., a fluoranthene and aflatoxin B1 degrading bacterium from contaminated soil of a former coal gas plant. Syst Appl Microbiol. 2004;27(6):653–60.

Article CAS Google Scholar

  79. Go JR, Wengenack NL, Abu Saleh OM, Corsini Campioli C, Deml SM, Wilson JW. Mycobacterium septicum: a 6-year clinical experience from a tertiary hospital and reference laboratory. J Clin Microbiol. 2020;58(12):e02091–20.

Article CAS Google Scholar

  80. Schinsky MF, McNeil MM, Whitney AM, Steigerwalt AG, Lasker BA, Floyd MM, et al. Mycobacterium septicum sp. nov., a new rapidly growing species associated with catheter-related bacteraemia. Int J Syst Evol Microbiol. 2000;50(2):575–81.

Article CAS Google Scholar

  81. Tsukamura M, van der Meulen HJ, Grabow WO. Numerical taxonomy of rapidly growing, scotochromogenic mycobacteria of the Mycobacterium parafortuitum complex: Mycobacterium austroafricanum sp. nov. and Mycobacterium diernhoferi sp. nov., nom. Rev. Int J Syst Evol Microbiol. 1983;33(3):460–9.

Google Scholar

  82. Konjek J, Souded S, Guerardel Y, Trivelli X, Bernut A, Kremer L, et al. Mycobacterium lutetiense sp. nov., Mycobacterium montmartrense sp. nov. and Mycobacterium arcueilense sp. nov., members of a novel group of non-pigmented rapidly growing mycobacteria recovered from a water distribution system. Int J Syst Evol Microbiol. 2016;66(9):3694–702.

Article CAS Google Scholar

  83. Awadh H, Mansour M, Shorman M. Bacteremia with an unusual pathogen: Mycobacterium neoaurum. Case Rep Infect Dis. 2016;2016:5167874.

Google Scholar

  84. Heckman GA, Hawkins C, Morris A, Burrows LL, Bergeron C. Rapidly progressive dementia due to Mycobacterium neoaurum meningoencephalitis. Emerg Infect Dis. 2004;10(5):924–7.

Article Google Scholar

  85. Gharbi R, Khanna V, Frigui W, Mhenni B, Brosch R, Mardassi H. Phenotypic and genomic hallmarks of a novel, potentially pathogenic rapidly growing Mycobacterium species related to the Mycobacterium fortuitum complex. Sci Rep. 2021;11(1):13011.

Article CAS Google Scholar

  86. Levasseur A, Asmar S, Robert C, Drancourt M. Draft genome sequence of Mycobacterium houstonense strain ATCC 49403T. Genome Announc. 2016;4(3):e00443–16.

Google Scholar

  87. Biet F, Boschiroli ML. Non-tuberculous mycobacterial infections of veterinary relevance. Res Vet Sci. 2014;97:S69–77.

Article Google Scholar

  88. Weitzman I, Osadczyi D, Corrado M, Karp D. Mycobacterium thermoresistibile: a new pathogen for humans. J Clin Microbiol. 1981;14(5):593–5.

Article CAS Google Scholar

  89. Ruan J, Li X-Y, Chen H. Mycobacterium chubuense hand infection. IDCases. 2020;20:e00742.

Article Google Scholar

  90. Tanaka S, Hoshino Y, Sakagami T, Fukano H, Matsui Y, Hiranuma O. Pathogenicity of Mycolicibacterium phlei, a non-pathogenic nontuberculous mycobacterium in an immunocompetent host carrying anti-interferon gamma autoantibodies: a case report. BMC Infect Dis. 2019;19(1):1–6.

Article Google Scholar

  91. Deinhardt-Emmer S, Höring S, Mura C, Hillemann D, Hermann B, Sachse S, et al. First time isolation of Mycobacterium hassiacum from a respiratory sample. Clin Med Insights Circul Respir Pulmon Med. 2018;12:1179548417747529.

Google Scholar

  92. Salzer HJF, Chitechi B, Hillemann D, Mandl M, Paar C, Mitterhumer M, et al. Nontuberculous mycobacterial pulmonary disease from Mycobacterium hassiacum, Austria. Emerg Infect Dis. 2020;26(11):2776–8.

Article CAS Google Scholar

  93. Sood S, Yadav A, Shrivastava R. Mycobacterium aurum is unable to survive Mycobacterium tuberculosis latency associated stress conditions: implications as non-suitable model organism. Indian J Microbiol. 2016;56(2):198–204.

Article CAS Google Scholar

  94. Tsukamura M. Mycobacterium chitae: a new species. Japanese J Microbiol. 1967;11(1):43–7. https://doi.org/10.1111/j.1348-0421.1967.tb00319.x.

Article CAS Google Scholar

  95. Liu B, Zheng D, Jin Q, Chen L, Yang J. VFDB 2019: a comparative pathogenomic platform with an interactive web interface. Nucleic Acids Res. 2019;47(D1):D687–92.

Article CAS Google Scholar

  96. Rivera-Calzada A, Famelis N, Llorca O, Geibel S. Type VII secretion systems: structure, functions and transport models. Nat Rev Microbiol. 2021;19(9):567–84.

Article CAS Google Scholar

  97. Be NA, Lamichhane G, Grosset J, Tyagi S, Cheng Q-J, Kim KS, et al. Murine model to study the invasion and survival of Mycobacterium tuberculosis in the central nervous system. J Infect Dis. 2008;198(10):1520–8.

Article Google Scholar

  98. van Leeuwen LM, Boot M, Kuijl C, Picavet DI, van Stempvoort G, van der Pol SM, et al. Mycobacteria employ two different mechanisms to cross the blood–brain barrier. Cell Microbiol. 2018;20(9):e12858.

Article Google Scholar

  99. Sala A, Bordes P, Genevaux P. Multiple toxin-antitoxin systems in Mycobacterium tuberculosis. Toxins. 2014;6(3):1002–20.

Article Google Scholar

  100. Keren I, Minami S, Rubin E, Lewis K. Characterization and transcriptome analysis of Mycobacterium tuberculosis persisters. MBio. 2011;2(3):e00100–11.

Article Google Scholar

  101. Cater JC, Redmond WB. Mycobacterial phages isolated from stool specimens of patients with pulmonary disease. Am Rev Respir Dis. 1963;87(5):726–9.

CAS Google Scholar

  102. Huh H, Wong S, Jean JS, Slavcev R. Bacteriophage interactions with mammalian tissue: therapeutic applications. Adv Drug Deliv Rev. 2019;145:4–17.

Article CAS Google Scholar

  103. Glickman C, Kammlade SM, Hasan NA, Epperson LE, Davidson RM, Strong M. Characterization of integrated prophages within diverse species of clinical nontuberculous mycobacteria. Virol J. 2020;17(1):1–13.

Article Google Scholar

  104. Sassi M, Gouret P, Chabrol O, Pontarotti P, Drancourt M. Mycobacteriophage-drived diversification of Mycobacterium abscessus. Biol Direct. 2014;9(1):1–15.

Article CAS Google Scholar

  105. Scheidegger S. Mittelalterliche Quecksilbervergiftungen. In: Dhom G, ed. Verhandlungen der Deutschen Gesellschaft für Pathologie. Stuttgart: Gustav Fisher Verlag. 1977;61:374.

  106. Wujastyk D. Histories of mercury in medicine across Asia and beyond. Asiatische Studien-Études Asiatiques. 2015;69(4):819–30.

Article Google Scholar

  107. Bachour N. Healing with mercury: the uses of mercury in Arabic medical literature. Asiatische Studien-Études Asiatiques. 2015;69(4):831–66.

Article Google Scholar

  108. O’Shea JG. ‘Two minutes with venus, two years with mercury’--mercury as an antisyphilitic chemotherapeutic agent. J R Soc Med. 1990;83(6):392–5.

Article Google Scholar

  109. Norn S, Permin H, Kruse E, Kruse PR. Mercury--a major agent in the history of medicine and alchemy. Dansk medicinhistorisk arbog. 2008;36:21–40.

Google Scholar

  110. Fornaciari A, Chericoni S, Stefanelli F, Fornaciari G, Giuffra V. Renaissance mercurial therapy in the mummies of saint Domenico Maggiore in Naples: a palaeopathological and palaeotoxicological approach. Archaeol Anthropol Sci. 2022;14(3):1–13.

Article Google Scholar

  111. Gad SC, Pham T. Mercury. In: Wexler P, editor. Encyclopedia of toxicology (third edition). Oxford: Academic; 2014. p. 207–10.

Chapter Google Scholar

  112. Park J-D, Zheng W. Human exposure and health effects of inorganic and elemental mercury. J Prev Med Publ Health = Yebang Uihakhoe chi. 2012;45(6):344–52.

Article Google Scholar

  113. Friberg L, Mottet NK. Accumulation of methylmercury and inorganic mercury in the brain. Biol Trace Elem Res. 1989;21(1):201–6.

Article CAS Google Scholar

  114. Liu J, Cui J, Wei X, Li W, Liu C, Li X, et al. Investigation on selenium and mercury interactions and the distribution patterns in mice organs with LA-ICP-MS imaging. Anal Chim Acta. 2021;1182:338941.

Article CAS Google Scholar

  115. LaFond RE, Lukehart SA. Biological basis for syphilis. Clin Microbiol Rev. 2006;19(1):29–49.

Article CAS Google Scholar

  116. Park K-H, Lee MS, Hong IK, Sung J-Y, Choi S-H, Park SO, et al. Bone involvement in secondary syphilis: a case report and systematic review of the literature. Sex Transm Dis. 2014;41(9):532–7.

Article Google Scholar

  117. Çakmak SK, Tamer E, Karadağ AS, Waugh M. Syphilis: a great imitator. Clin Dermatol. 2019;37(3):182–91.

Article Google Scholar

  118. Schütz J. Ueber Bacillen bei syphilis. DMW-Deutsche Medizinische Wochenschrift. 1885;11(19):320–1.

Article Google Scholar

  119. Lustgarten S. The bacillus of syphilis. Lancet. 1885;125(3214):609–10.

Article Google Scholar

  120. Bos KI, Jäger G, Schuenemann VJ, Vågene ÅJ, Spyrou MA, Herbig A, et al. Parallel detection of ancient pathogens via array-based DNA capture. Philos Transact Royal Soc B: Biol Sci. 2015;370(1660):20130375.

Article Google Scholar

  121. Ratnatunga CN, Lutzky VP, Kupz A, Doolan DL, Reid DW, Field M, et al. The rise of non-tuberculosis mycobacterial lung disease. Front Immunol. 2020;11. https://www.frontiersin.org/articles/10.3389/fimmu.2020.00303/full.

  122. Ahmed I, Tiberi S, Farooqi J, Jabeen K, Yeboah-Manu D, Migliori GB, et al. Non-tuberculous mycobacterial infections—a neglected and emerging problem. Int J Infect Dis. 2020;92:S46–50.

Article CAS Google Scholar

  123. Desai AN, Hurtado R. Nontuberculous mycobacterial infections. JAMA. 2021;325(15):1574.

Article Google Scholar

  124. Fellows Yates JA, Velsko IM, Aron F, Posth C, Hofman CA, Austin RM, et al. The evolution and changing ecology of the African hominid oral microbiome. Proc Natl Acad Sci. 2021;118(20):e2021655118.

Article CAS Google Scholar

  125. Maixner F, Mitterer C, Jäger HY, Sarhan MS, Valverde G, Lücker S, et al. Linear polyacrylamide is highly efficient in precipitating and purifying environmental and ancient DNA. Methods Ecol Evol. 2021;n/a(n/a). https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13772.

  126. Meyer M, Kircher M. Illumina sequencing library preparation for highly multiplexed target capture and sequencing. Cold Spring Harb Protoc. 2010;2010(6):pdb prot5448.

Article Google Scholar

  127. Chen S, Zhou Y, Chen Y, Gu J. Fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34(17):i884–90.

Article Google Scholar

  128. Shen W, Le S, Li Y, Hu F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. PLoS One. 2016;11(10):e0163962.

Article Google Scholar

  129. Rosenbloom KR, Armstrong J, Barber GP, Casper J, Clawson H, Diekhans M, et al. The UCSC genome browser database: 2015 update. Nucleic Acids Res. 2015;43(Database issue):D670–81.

Article CAS Google Scholar

  130. Andrews RM, Kubacka I, Chinnery PF, Lightowlers RN, Turnbull DM, Howell N. Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. Nat Genet. 1999;23(2):147.

Article CAS Google Scholar

  131. Li H, Durbin R. Fast and accurate long-read alignment with Burrows-wheeler transform. Bioinformatics. 2010;26(5):589–95.

Article Google Scholar

  132. Okonechnikov K, Conesa A, Garcia-Alcalde F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Bioinformatics. 2016;32(2):292–4.

CAS Google Scholar

  133. Renaud G, Slon V, Duggan AT, Kelso J. Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA. Genome Biol. 2015;16:224.

Article Google Scholar

  134. Weissensteiner H, Pacher D, Kloss-Brandstatter A, Forer L, Specht G, Bandelt HJ, et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic Acids Res. 2016;44(W1):W58–63.

Article CAS Google Scholar

  135. Skoglund P, Storå J, Götherström A, Jakobsson M. Accurate sex identification of ancient human remains using DNA shotgun sequencing. J Archaeol Sci. 2013;40:4477–82.

Article CAS Google Scholar

  136. Buchfink B, Reuter K, Drost H-G. Sensitive protein alignments at tree-of-life scale using DIAMOND. Nat Methods. 2021;18(4):366–8.

Article CAS Google Scholar

  137. Coordinators NR. Database resources of the national center for biotechnology information. Nucleic Acids Res. 2014;42(D1):D7–D17.

Article Google Scholar

  138. Wang Z-P, Xing H-L, Dong L, Zhang H-Y, Han C-Y, Wang X-C, et al. Egg cell-specific promoter-controlled CRISPR/Cas9 efficiently generates homozygous mutants for multiple target genes in Arabidopsis in a single generation. Genome Biol. 2015;16(1):144.

Article Google Scholar

  139. Nurk S, Meleshko D, Korobeynikov A, Pevzner PA. metaSPAdes: a new versatile metagenomic assembler. Genome Res. 2017;27(5):824–34.

Article CAS Google Scholar

  140. Kang DD, Li F, Kirton E, Thomas A, Egan R, An H, et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. PeerJ. 2019;7:e7359.

Article Google Scholar

  141. Wu YW, Singer SW. Recovering individual genomes from metagenomes using MaxBin 2.0. Curr Protoc. 2021;1(5):e128.

Article CAS Google Scholar

  142. Alneberg J, Bjarnason BS, de Bruijn I, Schirmer M, Quick J, Ijaz UZ, et al. Binning metagenomic contigs by coverage and composition. Nat Methods. 2014;11(11):1144–6.

Article CAS Google Scholar

  143. Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Res. 2015;25(7):1043–55.

Article CAS Google Scholar

  144. Zhou C, Xu Q, He S, Ye W, Cao R, Wang P, et al. GTDB: an integrated resource for glycosyltransferase sequences and annotations. Database (Oxford). 2020;2020. https://academic.oup.com/database/article/doi/10.1093/database/baaa047/5857526.

  145. Chaumeil PA, Mussig AJ, Hugenholtz P, Parks DH. GTDB-Tk: a toolkit to classify genomes with the genome taxonomy database. Bioinformatics. 2019. https://academic.oup.com/bioinformatics/article/36/6/1925/5626182.

  146. Swindell SR, Plasterer TN. Seqman. In: _Sequence data analysis guidebook_ : Springer; 1997. p. 75–89.

Chapter Google Scholar

  147. Asnicar F, Thomas AM, Beghini F, Mengoni C, Manara S, Manghi P, et al. Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0. Nat Commun. 2020;11(1):2500.

Article CAS Google Scholar

  148. Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80.

Article CAS Google Scholar

  149. Capella-Gutiérrez S, Silla-Martínez JM. Gabaldón T: trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009;25(15):1972–3.

Article Google Scholar

  150. Price MN, Dehal PS, Arkin AP. FastTree 2–approximately maximum-likelihood trees for large alignments. PLoS One. 2010;5(3):e9490.

Article Google Scholar

  151. Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014;30(9):1312–3.

Article CAS Google Scholar

  152. Langmead B, Salzberg SL. Fast gapped-read alignment with bowtie 2. Nat Methods. 2012;9:357.

Article CAS Google Scholar

  153. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The sequence alignment/map format and SAMtools. Bioinformatics. 2009;25(16):2078–9.

Article Google Scholar

  154. Seemann T. Prokka: rapid prokaryotic genome annotation. Bioinformatics. 2014;30(14):2068–9.

Article CAS Google Scholar

  155. Hyatt D, Chen G-L, LoCascio PF, Land ML, Larimer FW, Hauser LJ. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics. 2010;11(1):119.

Article Google Scholar

  156. Lagesen K, Hallin P, Rødland EA, Stærfeldt H-H, Rognes T, Ussery DW. RNAmmer: consistent and rapid annotation of ribosomal RNA genes. Nucleic Acids Res. 2007;35(9):3100–8.

Article CAS Google Scholar

  157. Gilchrist CL, Chooi Y-H. Clinker & clustermap. Js: automatic generation of gene cluster comparison figures. Bioinformatics. 2021;37(16):2473–5.

Article CAS Google Scholar

  158. Emms DM, Kelly S. OrthoFinder: phylogenetic orthology inference for comparative genomics. Genome Biol. 2019;20(1):1–14.

Article Google Scholar

  159. Xie Y, Wei Y, Shen Y, Li X, Zhou H, Tai C, et al. TADB 2.0: an updated database of bacterial type II toxin–antitoxin loci. Nucleic Acids Res. 2017;46(D1):D749–53.

Article Google Scholar

  160. Arndt D, Grant JR, Marcu A, Sajed T, Pon A, Liang Y, et al. PHASTER: a better, faster version of the PHAST phage search tool. Nucleic Acids Res. 2016;44(W1):W16–21.

Article CAS Google Scholar

  161. Nayfach S, Camargo AP, Schulz F, Eloe-Fadrosh E, Roux S. Kyrpides NC: CheckV assesses the quality and completeness of metagenome-assembled viral genomes. Nat Biotechnol. 2021;39(5):578–85.

Article CAS Google Scholar

  162. Ruiz-Perez CA, Conrad RE, Konstantinidis KT. MicrobeAnnotator: a user-friendly, comprehensive functional annotation pipeline for microbial genomes. BMC Bioinformatics. 2021;22(1):1–16.

Article Google Scholar

  163. Steinegger M, Söding J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Nat Biotechnol. 2017;35(11):1026–8.

Article CAS Google Scholar

  164. Roux S, Páez-Espino D, Chen I-MA, Palaniappan K, Ratner A, Chu K, et al. IMG/VR v3: an integrated ecological and evolutionary framework for interrogating genomes of uncultivated viruses. Nucleic Acids Res. 2021;49(D1):D764–75.

Article CAS Google Scholar

  165. Sequencing data of the mummified Lady from Barfüsser church from Basel, Switzerland. ENA accession PRJEB44723. https://www.ebi.ac.uk/ena/browser/view/PRJEB44723 (2022)




Download references

## Acknowledgements

We thank the Citizen Science Team of Basel (CSB) for the genealogical studies and the research in the City Town archives, which led to the identification of the Mummy. We also thank the CSB-team of genealogists of CSB, in particular Diana Gysin, Marie-Louise Gamma, Odette Haas, Marina Zulauf-Semmler, and Jürgen Rauber and the CSB-team of medical history research in the town ship archives of Basel, in particular Ursula Hirter-Trüb, Erika Borner, Regine Dendler, Dascha Herber, Ludwig Huber, Hans-Ulrich Fiechter, Brigitte Kuhn, Christof Meissburger, Gitta Reinhardt-Fehrenbach, Willy Ruess, Martin Schneider, Albert Spycher, and Helena Vogler. We thank Lena Granehäll for her assistance with processing the samples. We are grateful to the organizing team of Uppsala University “Snakemake bring-your-own-code (BYOC)” workshop for assisting in improving the metagenomic analysis workflow, and to the members of SPAAM community who helped during the peer-reviewing of the article.

## Funding

Support was provided by the European Regional Development Fund 2014-2020_CALL-FESR 2017 Research and Innovation_Autonomous Province of Bolzano - South Tyrol_Project: FESR1078-MummyLabs. The authors thank the Department of Innovation, Research and University of the Autonomous Province of Bozen/Bolzano for covering the Open Access publication costs.

## Author information

### Authors and Affiliations

  1. Eurac Research – Institute for Mummy Studies, 39100, Bolzano, Italy

Mohamed S. Sarhan, Christina Wurst, Albert Zink & Frank Maixner

  2. Institute of Medical Genetics and Pathology, University Hospital Basel, University of Basel, 4031, Basel, Switzerland

Alexandar Tzankov

  3. Department of Allergology, University Hospital Basel, 4031, Basel, Switzerland

Andreas J. Bircher

  4. Faculty of Biomedical Sciences, Università della Svizzera italiana, Lugano, Switzerland

Andreas J. Bircher

  5. Department of Biomedical Engineering, Institute of Forensic Medicine, University of Basel, 4056, Basel, Switzerland

Holger Wittig

  6. Citizen Science Basel; formerly Institute of Forensic Medicine, Forensic Chemistry and Toxicology, University of Basel, 4056, Basel, Switzerland

Thomas Briellmann

  7. University Center of Legal Medicine, Lausanne, Geneva, Switzerland

Marc Augsburger

  8. Natural History Museum Basel, 4051, Basel, Switzerland

Gerhard Hotz

  9. Integrative Prehistory and Archaeological Science, University of Basel, 4056, Basel, Switzerland

Gerhard Hotz




Authors

  1. Mohamed S. Sarhan

View author publications

You can also search for this author inPubMed Google Scholar

  2. Christina Wurst

View author publications

You can also search for this author inPubMed Google Scholar

  3. Alexandar Tzankov

View author publications

You can also search for this author inPubMed Google Scholar

  4. Andreas J. Bircher

View author publications

You can also search for this author inPubMed Google Scholar

  5. Holger Wittig

View author publications

You can also search for this author inPubMed Google Scholar

  6. Thomas Briellmann

View author publications

You can also search for this author inPubMed Google Scholar

  7. Marc Augsburger

View author publications

You can also search for this author inPubMed Google Scholar

  8. Gerhard Hotz

View author publications

You can also search for this author inPubMed Google Scholar

  9. Albert Zink

View author publications

You can also search for this author inPubMed Google Scholar

  10. Frank Maixner

View author publications

You can also search for this author inPubMed Google Scholar




### Contributions

MSS and FM designed the study and performed the analysis. MSS processed the samples, analyzed and visualized the data, and drafted manuscript. FM coordinated the molecular investigations. CW revised the human DNA analysis. AZ, TB, AJB, and GH were involved in the sampling campaign. AT and AJB performed the histopathological analyses. HW was responsible for the computed tomography imaging. MA and TB were responsible for the toxicological analyses. FM and AZ were responsible for the funding acquisition. GH was responsible for the curation of the mummy and was directing all anthropological, genealogical, and medical history research. MSS, CW, AZ, TB, AJB, GH, AZ, and FM edited the manuscript. All authors read and approved the final manuscript.

### Corresponding authors

Correspondence to Mohamed S. Sarhan or Frank Maixner.

## Ethics declarations

### Ethics approval and consent to participate

All human samples used in this study are more than 70 years old and anonymous. Therefore, ethical approval for the genetic analysis is not required under current Swiss law (https://www.admin.ch/opc/de/classified-compilation/20061313/index.html), but ethical guidelines were considered during sampling and storage, following the recommendations of the International Council of Museums (ICOM). The mummy is housed at the Natural History Museum of Basel (NHM), Switzerland, and the sample collection for this study has been performed in the framework of a scientific collaboration. The museum fully supported the study, and the curator is included as coauthor in the study. The consent is available upon request.

### Consent for publication

Not applicable.

### Competing interests

The authors declare no competing interests.

## Additional information

### Publisher’s Note

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Supplementary Information

### **Additional file 1: Table S1.**

Sequencing basic statistics and mitochondrial haplogroup assignments. **Table S2.** Taxonomic classification on the bacterial species-level as estimated by mapping the metagenomic reads against the NCBI-nr database using DIAMOND/BLASTx mode and lowest common ancestor (LCA) assignment using MEGAN. **Table S3.** Basic assembly statistics and CheckM results of different assembly approaches. **Table S4.** Comparison of total number of bacterial-assigned reads, as estimated using DIAMOND search against NCBI-nr database and Brain_NTM-assigned reads by mapping. **Table S5.** The lengths, coverage, and GC content of the contigs of the brain NTM after combining SPAdes and MEGAHIT assemblies. **Table S6.** Metadata on the characterized members of the genus _Mycolicibacterium_. **Table S7.** Combined annotation of the contig_38. **Table S8.** CheckV results of the contig_38.

### **Additional file 2: Figure S1.**

NCBI taxonomy-based cladogram of the metagenomic reads mapped to the reference genome of _Treponema pallidum_ subsp. _pallidum_ (NZ_CP010561.1). **Figure S2.** Top bacterial families in the brain tissue as inferred by different taxonomic classifiers. **Figure S3.** Read lengths distribution of human DNA of different tissues as well as the brain NTM. **Figure S4.** Description of the analyzed samples. **Figure S5.** Virulence genes of the brain NTM. **Figure S6.** Toxin/Antitoxin (TA) Systems in the analyzed mycobacterial genomes. **Figure S7.** Metagenomic binning of Anna Catharina Bischoff’s (ACB) brain sample. **Figure S8.** Correlation analysis between concentrations of mercury (Hg) and Selenium (Se) in different body parts.

## Rights and permissions

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated in a credit line to the data.

Reprints and permissions

## About this article

### Cite this article

Sarhan, M.S., Wurst, C., Tzankov, A. _et al._ A nontuberculous mycobacterium could solve the mystery of the lady from the Franciscan church in Basel, Switzerland. _BMC Biol_ **21** , 9 (2023). https://doi.org/10.1186/s12915-022-01509-7

Download citation

  * Received: 07 March 2022

  * Accepted: 15 December 2022

  * Published: 07 February 2023

  * DOI: https://doi.org/10.1186/s12915-022-01509-7




### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative 

### Keywords

  * Nontuberculous mycobacteria (NTM)
  * Ancient DNA (aDNA)
  * Bacteriophage
  * Syphilis
  * Brain infections
  * Mycobacteriosis
  * Franciscan church mummy
  * Anna Catharina Bischoff (ACB)
  * De novo assembly



Download PDF

Advertisement

####  BMC Biology

ISSN: 1741-7007

#### Contact us

  * General enquiries: journalsubmissions@springernature.com



  * Read more on our blogs
  * Receive BMC newsletters
  * Manage article alerts
  * Language editing for authors
  * Scientific editing for authors


  * Policies
  * Accessibility
  * Press center


  * Support and Contact
  * Leave feedback
  * Careers



### Follow BMC

  * BMC Twitter page
  * BMC Facebook page
  * BMC Weibo page



By using this website, you agree to our Terms and Conditions, Your US state privacy rights, Privacy statement and Cookies policy. Your privacy choices/Manage cookies we use in the preference centre. 

© 2025 BioMed Central Ltd unless otherwise stated. Part of Springer Nature. 
  *[DOI]: Digital Object Identifier
